yarn-project/cli-wallet/src/storage/wallet_db.ts

```diff
@@ -10,13 +10,15 @@ export const Aliases = ['accounts', 'contracts', 'artifacts', 'secrets', 'transa
 export type AliasType = (typeof Aliases)[number];
 
 export class WalletDB {
-  #accounts!: AztecMap<string, Buffer>;
-  #aliases!: AztecMap<string, Buffer>;
-  #bridgedFeeJuice!: AztecMap<string, Buffer>;
-  #transactions!: AztecMap<string, Buffer>;
+  #accounts!: AztecAsyncMap<string, Buffer>;
+  #aliases!: AztecAsyncMap<string, Buffer>;
+  #bridgedFeeJuice!: AztecAsyncMap<string, Buffer>;
+  #transactions!: AztecAsyncMap<string, Buffer>;
 
   private static instance: WalletDB;
 
+  private aliasCache = new Map<string, string>();
```

Comment:
This cache is necessary because the parsers for custom Commander options need to by synchronous.